logic/office.py

```diff
@@ -0,0 +1,82 @@
+# -*- coding: utf-8 -*-
+from models.employee import Employee
+
+
+def get_all_offices():
+    return [
+        row.office
+        for row in Employee.query(projection=['office'], distinct=True).fetch()
+        if row.office
+    ]
+
+
+def get_all_offices_compressed():
```

Comment:
Okay I will update it

Reply:
Sorry I did not get it. What Should I change?

What about leaving this at it is right now and after that we can update this part to make the import extensible and may be adding a separate table for departments and offices

Reply:
Yeah we can do that. Going forward we should make this configurable, since the code you're adding for filtering and matching office names seems highly specific to our internal use case.